tests/main/cgroup-freezer/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: Each snap process is moved to appropriate freezer cgroup
+details: |
+    This test creates a snap process that suspends itself and ensures that it
+    placed into the appropriate hierarchy under the freezer cgroup.
+prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local test-snapd-sh
+execute: |
+    # Start a "sleep" process in the background
+    test-snapd-sh -c 'exec sleep 1h' &
+    pid1=$!
+    # Give snap-confine a moment to start and perform its task.
```

Comment:
Lets write a stamp file in the sh -c, this way we can avoid the sleep and just busy-loop and wait for the file to appear.

Reply:
Aye, I'll change this in the morning, thank you for the suggestion.

Something like this?
```
touch task-1.stamp
exec sleep 1h
```